Modules/_codecsmodule.c

```diff
@@ -979,6 +979,26 @@ _codecs_register_error_impl(PyObject *module, const char *errors,
     Py_RETURN_NONE;
 }
 
+/*[clinic input]
+_codecs.unregister_error -> bool
```

Comment:
I would prefer to add a private undocumented function instead. Outside tests, I'm not sure that it's a common need to unregister an error handler.

Reply:
I'm ok with this!